monitoring/interpod.go

```diff
@@ -231,7 +232,7 @@ func waitTimeoutForPodRunningInNamespace(client *kube.Clientset, podName string,
 func waitForPodCondition(client *kube.Clientset, ns, podName, desc string, timeout time.Duration, condition podCondition) error {
 	log.Infof("waiting up to %v for pod %s status to be %s", timeout, podName, desc)
 	for start := time.Now(); time.Since(start) < timeout; time.Sleep(pollInterval) {
-		pod, err := client.CoreV1().Pods(ns).Get(podName, metav1.GetOptions{})
+		pod, err := client.CoreV1().Pods(ns).Get(context.TODO(), podName, metav1.GetOptions{})
```

Comment:
Should this be related to `timeout - time.Since(start)`?  I haven't looked at the far side of the call, but it seems like we wouldn't want it hanging past the timeout.